cell/src/main/java/jetbrains/jetpad/cell/Cell.java

```diff
@@ -70,6 +70,7 @@
   public static final CellPropertySpec<Boolean> HAS_SHADOW = new CellPropertySpec<>("hasShadow", false);
   public static final CellPropertySpec<Boolean> RED_UNDERLINE = new CellPropertySpec<>("redUnderline", false);
   public static final CellPropertySpec<Boolean> YELLOW_UNDERLINE = new CellPropertySpec<>("yellowUnderline", false);
+  public static final CellPropertySpec<Boolean> YELLOWISH_BACKGROUND = new CellPropertySpec<>("yellowishBackground", false);
```

Comment:
Should be named WARNING_BACKGROUND instead of YELLOWISH_BACKGROUND, 
YELLOW_UNDERLINE -> WARNING_UNDERLINE, 
RED_UNDERLINE -> ERROR_UNDERLINE.